lib/normalize.js

```diff
@@ -264,7 +264,8 @@ const normalize = async (pkg, { strict, steps, root, changes, allowLegacyCase })
         }
       }
     } else {
-      changes?.push(`Removed invalid "scripts"`)
+      // it only makes sense to fix if package.json has scripts field
+      data.scripts && changes?.push(`Removed invalid "scripts"`)
```

Comment:
Let's make this an `if` statement

Reply:
Sure, I see some other scenarios as well that I'll cover in next changes

Reply:
These changes were not respecting false values like `null`, `0` or `''`. Instead modified top if condition to check if package has `scripts` then only do the updates.